### PR TITLE
feat(clearRefinements): implement InstantSearch.css

### PR DESCRIPTION
**Pull request by francoischalifour**
## Spec

https://instantsearch-css.netlify.com/widgets/clear-refinements/

## Stories

https://deploy-preview-3308--instantsearchjs.netlify.com/stories/?selectedStory=ClearRefinements.default

### CODE REVIEW

File: storybook/app/builtin/stories/clear-refinements.stories.js [hunk 1]
```diff
@@ -27,25 +27,43 @@ export default () => {
...
+            templates: {
+              resetLabel: 'Clear refinements and query',
+            },
           })
         );
       })
     )
     .add(
-      'with clear refinements and query',
+      'with refinements and query',
```
Comment (Haroenv) [inline storybook/app/builtin/stories/clear-refinements.stories.js]: this is duplicate right?

**Comment by francoischalifour** [comment]
I force-pushed to get the CI to work on this branch (d900b5878525c666ffb41ab4ee0d51840c9b90b4).

File: storybook/app/builtin/stories/clear-refinements.stories.js (hunk 1 above)
Comment (francoischalifour) [reply storybook/app/builtin/stories/clear-refinements.stories.js]: Duplicate with which one? I updated the stories anyway.

File: src/lib/__tests__/utils-test.js [hunk 2]
```diff
@@ -1007,202 +1007,89 @@ describe('utils.clearRefinements', () => {
     return helper;
   };
 
-  describe('Without clearsQuery', () => {
```
Comment (francoischalifour) [inline src/lib/__tests__/utils-test.js]: Bunch of tests have been removed because the function (and its signature) has been simplified.

**Pull request merged**